android/android_browser_tests.gni

```diff
@@ -106,6 +106,7 @@ android_test_exception_deps = [
   "//brave/browser/test:browser_tests",
   "//brave/browser/ui:browser_tests",
   "//brave/browser/ui/tabs/test:browser_tests",
+  "//brave/browser/ui/toolbar:brave_app_menu_browser_tests",
```

Comment:
I think what we want here are tests that should run on android, but need to be converted from InProcessBrowserTest so this should probably just go in an `!is_android` block in the test target since it's testing desktop-only code?

Reply:
cc: @AlexeyBarabash

Reply:
It is fixed by 1bd307195cb267efef5bc9a0d20f77f69a206150 and d853aafafd26749b19aaec19a0609198d1c75572 (revert)